hphp/runtime/vm/jit/prof-data.h

```diff
@@ -358,8 +358,10 @@ struct ProfData {
   int64_t transCounter(TransID id) const {
     folly::SharedMutex::ReadHolder lock{m_transLock};
     assertx(id < m_transRecs.size());
-    auto const counter = m_counters.get(id);
+    auto counter = m_counters.get(id);
     auto const initVal = m_counters.getDefault();
+    if (RuntimeOption::EvalJitPGOFastProfiling &&
+        (counter > initVal)) counter = 0;
```

Comment:
You have a tab here.  Please try to convert tabs to spaces on all files before putting up a PR (or maybe configure your editor to always use spaces).

Reply:
@ottoni: I don't see how this can happen. Even with racy decrements, counter should never be larger than initVal.